docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md

```diff
@@ -62,6 +62,6 @@ DETECTION_THRESHOLD=0.50
 ```
 
 ## Rebuild and Run Pipeline
-1. Rebuild the python app and ovms-client: `make build-python-apps`
+1. Rebuild the python app and profile-launcher: `make build-python-apps`
```

Comment:
we need to update the prerequisites to include how to install Golang on localhost. It's required to build the profile-launcher.

Reply:
that is a good point; i'll add that

Reply:
i [added here](https://github.com/jim-wang-intel/vision-self-checkout/blob/remove-ovms-client-docker/docs_src/OVMS/pipelinesetup.md#step-2-install-golang-120)